### PR TITLE
don't scroll to top if it is staff

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity#22366

*Description of changes:*
Stop scrolling to top to review if the user is staff

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -1185,7 +1185,7 @@ export default defineComponent({
           ? await getMHRegistrationSummary(getMhrInformation.value.mhrNumber, false)
           : null
 
-        if (!!regSum && !!regSum.lienRegistrationType) {
+          if (!!regSum && !!regSum.lienRegistrationType && !isRoleStaffReg.value) {
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Looks good but looks like indent/spacing is slightly off

**Review by cameron-eyds — APPROVED** [review]
LG! Just a small indent to fix

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (flutistar) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Thanks @cameron-eyds , I removed the indent.
I am gonna merge the PR now.

**Comment by flutistar** [comment]
Fixed indent issue. 
@cameron-eyds  FYI.

**Pull request merged**